readthedocs/proxito/views/hosting.py

```diff
@@ -406,36 +354,6 @@ def _v1(self, project, version, build, filename, url, request):
 
         data = {
             "api_version": "1",
-            "projects": {
```

Comment:
I worry a little bit about deleting this on deploy, since it will lead to a timing issue between the client & server. I'd probably just add the new fields to the existing API response, so we can test that appraoch without breaking old API clients during deploy?

Reply:
Yeah, it makes sense to deploy everything needed in the backend first without removing the old pattern 👍🏼